### ISSUE TITLE
[BUG] Missing mandatory rules for administracion_efectivo.ear binary analysis

### ISSUE THREAD

**Issue by mguetta1** [konveyor/analyzer-lsp#906]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Konveyor version

main

### Priority

Major

### Current Behavior

The issue was tested with `deisable_maven_search: true`
Please see CI run: https://github.com/konveyor/go-konveyor-tests/actions/runs/17824677012/job/50674965749?pr=332#step:13:2312

The following rules are missing:
- local-storage-00001
- local-storage-00002

The binary app can be downloaded from: https://github.com/konveyor/go-konveyor-tests/tree/main/analysis/data/binary

### How Reproducible

Always (Default)

### Environment

```markdown

```

**Comment by mguetta1** [comment]
Hi @shawn-hurley, thanks for looking into this!
- The scope of the analysis is binary artifact uploaded from the user's machine (can be downloaded from [here](https://github.com/konveyor/go-konveyor-tests/tree/main/analysis/data/binary))
- label selector: `konveyor.io/target=cloud-readiness`
- The expected violations are listed [here](https://github.com/konveyor/go-konveyor-tests/blob/main/analysis/tc_administracion_efectivo_upload_binary.go#L26-L75)

**Comment by mguetta1** [comment]
@shawn-hurley looks like the "expected" incidents are all from open-source libs and they should not be found as violations unless `withKnownLibs` flag is on.
With that said, I am closing the issue.
